src/Tribe/Main.php

```diff
@@ -1468,15 +1469,32 @@ public function get_event_label_singular() {
 		 * @since ??
 		 * @since 6.0.0 Relies on √2 code.
 		 *
-		 * Disabling this behavior always is possible with:
+		 * Disabling this behavior completely is possible with:
 		 *
-		 *     add_filter( 'tribe_events_add_no_index_meta', '__return_false' );
+		 *     add_filter( 'tec_events_add_no_index_meta_tag', '__return_false' );
 		 *
-		 *  Enabling it for all event views is possible with:
+		 *  Always adding the noindex meta tag for all event views is possible with:
 		 *
 		 *     add_filter( 'tribe_events_add_no_index_meta', '__return_true' );
+		 *
+		 *  Always adding the noindex meta tag for a specific event view is possible with:
+		 *
+		 *     add_filter( "tribe_events_{$view}_add_no_index_meta", '__return_true' );
+		 *
+		 *  Where `$view` above is the view slug, e.g. `month`, `day`, `list`, etc.
 		 */
 		public function issue_noindex() {
+			global $wp_query;
```

Comment:
Excellent chance to refactor this out of `Main` into a specialized class.

Reply:
See `src/Events/SEO/No_Index.php`